### PR TITLE
Update the French translation file

### PR DESCRIPTION
**Pull request by lionel-m**
## Description
Update the French translation file. #3574 and #3569 are included.

### CODE REVIEW

**Comment by gkatsev** [comment]
Awesome, thanks!

File: lang/fr.json [hunk 1]
```diff
@@ -15,12 +15,26 @@
...
   "subtitles off": "Sous-titres désactivés",
-  "Captions": "Sous-titres",
-  "captions off": "Sous-titres désactivés",
+  "Captions": "Sous-titres transcrits",
+  "captions off": "Sous-titres transcrits désactivés",
   "Chapters": "Chapitres",
+  "Close Modal Dialog": "Fermer la boîte de dialogue modale",
+  "Descriptions": "Descriptions",
+  "descriptions off": "descriptions désactivées",
+  "Audio": "Audio",
```
Comment (gkatsev) [inline lang/fr.json]: We've actually decided that we only need `Audio Track` right now, so, this line is not necessary.

File: lang/fr.json [hunk 2]
```diff
@@ -15,12 +15,26 @@
...
+  "Audio Track": "Piste audio",
   "You aborted the media playback": "Vous avez interrompu la lecture de la vidéo.",
   "A network error caused the media download to fail part-way.": "Une erreur de réseau a interrompu le téléchargement de la vidéo.",
   "The media could not be loaded, either because the server or network failed or because the format is not supported.": "Cette vidéo n'a pas pu être chargée, soit parce que le serveur ou le réseau a échoué ou parce que le format n'est pas reconnu.",
   "The media playback was aborted due to a corruption problem or because the media used features your browser did not support.": "La lecture de la vidéo a été interrompue à cause d'un problème de corruption ou parce que la vidéo utilise des fonctionnalités non prises en charge par votre navigateur.",
   "No compatible source was found for this media.": "Aucune source compatible n'a été trouvée pour cette vidéo."
```
Comment (gkatsev) [inline lang/fr.json]: there's a comma missing at the end of this line.

**Comment by BrandonOCasey** [comment]
LGTM

**Comment by gkatsev** [comment]
LGTM

**Pull request closed without merging**